### ISSUE TITLE
Exception with two closures on the same line of code

### ISSUE THREAD

**Issue by sathvikp** [liebig/cron#78]
We have our code minified with [php-obfuscator](https://github.com/naneau/php-obfuscator), so the code is condensed to a single line on most files.

I'm getting the following exception when running a job, as the `$return` value in the `cron.jobError` event:
`Exception in job recurring_date_time_event_6: Two closures were declared on the same line of code. Cannot determine which closure was the intended target.`

Is this something to do with this Cron package or an effect of Laravel itself?

Using the latest cron version 1.1.0 with Laravel 4.2, on PHP 5.5.9-1ubuntu4.16

Thanks in advance!

**Comment by liebig** [comment]
I think the obfuscator produces this error. I have never seen a `Two closures were declared on the same line of code` error before and if the obfuscator condenses the code to a single line and the error is, that two closures were declared on the same line of code, the obfuscation seems to be the problem. I think it is not a problem with Laravel or the Cron library. Did you try to use the code without obfuscate it?

**Comment by sathvikp** [comment]
Thanks for the prompt response.

It works fine without the obfuscation. 

We're only obfuscating the app folder and the rest of the app seems to be working fine even with obfuscation ....

Is there a way we could get the full error stack trace and file/line that is causing this error from `cron.jobError`?

**Comment by sathvikp** [comment]
A helper class file had two closures and **the issue was fixed** when we moved them into two separate files. So this most likely was an issue with the obfuscator. 

Strangely the same obfuscated helper class has been working as expected when used anywhere else in the app - it only causes this issue when used with Cron (even with the URL based `/cron.php` route).

I'll be happy to help you investigate further if interested.

Thank you once again for all your help and support.
